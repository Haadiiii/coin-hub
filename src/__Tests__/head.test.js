import React from 'react';
import renderer from 'react-test-renderer';
import Head from './__mock__/head';

describe('Details', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Head />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
