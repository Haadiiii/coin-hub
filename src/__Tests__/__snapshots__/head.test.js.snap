// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details renders correctly 1`] = `
<div
  className="nav"
>
  <h3>
    COVID-19
  </h3>
  <div>
    <ul>
      <li>
        <img
          alt=""
          src="https://img.icons8.com/external-anggara-glyph-anggara-putra/32/FFFFFF/external-mic-basic-user-interface-anggara-glyph-anggara-putra.png"
        />
      </li>
      <li>
        <img
          alt=""
          src="https://img.icons8.com/external-anggara-glyph-anggara-putra/32/FFFFFF/external-setting-user-interface-basic-anggara-glyph-anggara-putra.png"
        />
      </li>
    </ul>
  </div>
</div>
`;
